fonts/Makefile.am

```diff
@@ -27,13 +27,15 @@ TTF_GREGORIO_OP = gregorio-op.ttf gregorio-op-hollow.ttf gregorio-op-hole.ttf
 TTF_GRANAPADANO = granapadano.ttf granapadano-hollow.ttf granapadano-hole.ttf
 TTF_GRANAPADANO_OP = granapadano-op.ttf granapadano-op-hollow.ttf granapadano-op-hole.ttf
 TTF_EXTRA = greextra.ttf gregall.ttf gresgmodern.ttf grelaon.ttf
-TTF_ALL = $(TTF_GRECILIAE) $(TTF_GRECILIAE_OP) $(TTF_GREGORIO) \
-		  $(TTF_GREGORIO_OP) $(TTF_GRANAPADANO) $(TTF_GRANAPADANO_OP)
-SOURCEFILES = gregorio-base.sfd granapadano-base.sfd greciliae-base.sfd \
-			  greextra.sfd gregall.sfd gresgmodern.sfd grelaon.sfd
+TTF_ALL = $(TTF_GRECILIAE) $(TTF_GRECILIAE_OP) $(TTF_GREGORIO) $(TTF_GREGORIO_OP) $(TTF_GRANAPADANO) $(TTF_GRANAPADANO_OP)
```

Comment:
I don't like these really long lines, but they work, so I'm not going to argue too hard.